src/data/analysis/IOTM Reviews/Chest Mimic.mdx

```diff
@@ -0,0 +1,59 @@
+
+Anyway. Having turned that footnote into two paragraphs of old-man rambling, let's cover the -real- value of the Chest Mimic -- **mimic eggs**. Whenever you have 50 familiar experience, you can cast a skill in combat (at a cost of 50 familiar XP) where you can ask your familiar to "lay an egg." This will give you a quest item. These are usable to generate a copy of whatever you used the skill on, allowing you to fight it again. You can fight 11 copies a day through this usage. That's cool! What's even cooler is that the familiar also opens a new zone on the Right Side of the Tracks -- the DNA Bank. There, you can do one of two things:
+
+- "Donate" an egg for science (three times per day)
+- "Extract" an egg into any monster that science can generate (essentially a fax-like summon; can be done eleven times per day, on a shared counter with simply using the eggs, for 100 total familiar XP)
+
+What does that mean, exactly? Well. Any monster that hits (globally) 100 eggs donated gets added to the pool of monsters that can be extracted at the DNA bank. [You can see the current progress on monster donations here.](https://semenar.am/kol/eggnet_monitor/) That means you can summon that monster at any time for 100 familiar experience. 
+
+So, to summarize: as long as you're already encountering the monster, you can copy the monster you're fighting for just 50 XP -- but if you want to summon up a brand new friend you can't naturally access, you need to spend 100. While that's a lot of XP (550 for 11 copies, 1100 for 11 faxes), it isn't actually as wild as it might sound at first glance; as I'll note in the synergies section, there are a few ways you can really decrease the time to accumulate all that experience, and this familiar will LOVE using freekills and freefights.
+
+## Speedrun Applicability
+
+Copies and faxes are super important parts of speedrunning. When you're trying to aggressively cut turns, you want to spend as few turns as possible generating the monsters you need to generate. Banishes, macrometeorite-likes, and free runs are good ways to accomplish this. But simply summoning the monster into existence saves the use of every single one of those, which is a massive boon to anyone with a need for speed. It also, amusingly, makes copies pretty hard to value. 
```

Comment:
These 3 paragraphs seem like they belong in the Faxes and Summons file.

Reply:
i see what you mean, but mildly disagree -- i think the intro of the Faxes & Summons page does a good enough job explaining their value and there is some value to reiterating why summons are so powerful in the chest mimic page, for readers who do not click through to the second tactics page. i'll keep it as is for now. (but respect the point!)

Reply:
actually, hm. maybe these paragraphs should be in both files. now i'm debating lol